sass/main.scss

```diff
@@ -94,12 +101,33 @@ p {
     font-size: 1.3em;
 }
 
+// Padding for all sections where needed
+// (modified on media queries only)
+.section-pad {
+    padding: 140px;
+}
+
+.section-pad-top {
+    padding-top: 140px;
+}
+
+.section-pad-left {
+    padding-left: 140px;
+}
+
+
 // Nav
 nav {
     background-color: rgba(white, 5%);
     display: flex;
     justify-content: space-between;
     font-size: $nav-fontsize;
+    height: $nav-height;
+    position: absolute;
```

Comment:
Switch to fixed per original design

Reply:
I'll check w/ May on this 'cause she didnt seem to mind on the staging site, so things may have changed

Reply:
Decision: fixed on desktop, not fixed on mobile. Coming in #1058